packages/public-header/src/public-header.tsx

```diff
@@ -121,7 +125,19 @@ export function PublicHeader({
         </Logo>
 
         <ExtraActionsContainer>
-          <ExtraActionItem href="/my-bookings">내 예약</ExtraActionItem>
+          <ExtraActionItem
+            href="#"
+            onClick={() => {
+              if (onClick) {
+                onClick()
+              } else {
+                document.location.href = '/my-bookings'
+              }
+              return false
+            }}
```

Comment:
```suggestion
            onClick={onClick}
```

그래서 onClick은 그냥 그대로 전달시켜주면 됩니다.